access-control-app/src/cmr/access_control/services/event_handler.clj

```diff
@@ -80,24 +80,28 @@
 (defmethod handle-indexing-event [:concept-delete :collection]
   [context {:keys [concept-id revision-id]}]
   (let [concept-map (mdb/get-concept context concept-id revision-id)
-        collection-concept (acl-matchers/add-acl-enforcement-fields-to-concept concept-map)
-        entry-title (:EntryTitle collection-concept)]
-    (doseq [acl-concept (acl-service/get-all-acl-concepts context)
-            :let [parsed-acl (acl-service/get-parsed-acl acl-concept)
-                  catalog-item-id (:catalog-item-identity parsed-acl)
-                  acl-entry-titles (:entry-titles (:collection-identifier catalog-item-id))]
-            :when (and (= (:provider-id collection-concept) (:provider-id catalog-item-id))
-                       (some #{entry-title} acl-entry-titles))]
-      (if (= 1 (count acl-entry-titles))
-        ;; the ACL only references the collection being deleted, and therefore the ACL should be deleted
-        (acl-service/delete-acl (transmit-config/with-echo-system-token context)
-                                (:concept-id acl-concept))
-        ;; otherwise the ACL references other collections, and will be updated
-        (let [new-acl (update-in parsed-acl
-                                 [:catalog-item-identity :collection-identifier :entry-titles]
-                                 #(remove #{entry-title} %))]
-          (acl-service/update-acl (transmit-config/with-echo-system-token context)
-                                  (:concept-id acl-concept) new-acl))))))
+        collection-concept (acl-matchers/add-acl-enforcement-fields-to-concept concept-map)]
+    (doseq [key-path [:entry-titles :concept-ids]]
```

Comment:
Thats how I had it at first, It didn't work that way during testing.  I will check it again.